lib/Horde/Core/VersionChecker.php

```diff
@@ -0,0 +1,485 @@
+<?php
+/**
+ * Copyright 2025 Horde LLC (http://www.horde.org/)
+ *
+ * See the enclosed file LICENSE for license information (LGPL). If you
+ * did not receive this file, see http://www.horde.org/licenses/lgpl21.
+ *
+ * @author   Torben Dannhauer <torben+horde@dannhauer.de>
+ * @category Horde
+ * @package  Core
+ */
+
+class Horde_Core_VersionChecker
```

Comment:
We shouldn't write new code in unnamespaced PSR-0 format

Reply:
Sure, will do if things are more stable. Have to figure out PSR-4 style.

Reply:
Changed to PSR-4. IKs the folder right?